### PR TITLE
automate asc for iot and sentinel 

### PR DESCRIPTION
**Pull request by monikavar**

### CODE REVIEW

File: deploy/automate-asc.ps1 [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+ .PARAMETER subscriptionId
+    Optional, the subscription id where resources will be deployed.
+#>
+
+#*******************************************************************************************************
+# Get Access token using AzureRM module
+#*******************************************************************************************************
+function Get-AzureRmCachedAccessToken()
+{
```
Comment (marcschier) [inline deploy/automate-asc.ps1]: Consisten curly braces, i think same line is good

**Review by marcschier — APPROVED** [review]

File: deploy/local/template.json [hunk 2]
```diff
@@ -201,7 +201,38 @@
             "metadata": {
                 "description": "The KeyVault SKU to use."
             }
-        }
+        },
+		"workspaceName": {
```
Comment (marcschier) [inline deploy/local/template.json]: Spaces instead of tabs

File: deploy/local/template.json [hunk 3]
```diff
@@ -432,6 +477,10 @@
         "iothub-consumer-group": {
             "type": "string",
             "value": "[variables('onboardingConsumerGroupName')]"
+        },
+		"workspace-name": {
```
Comment (marcschier) [inline deploy/local/template.json]: Spaces instead of tabs.

**Review by marcschier — CHANGES_REQUESTED** [review]
Could you also add this to the other deployments?

**Comment by monikavar** [comment]
Script for automating asc for iot, workspace creation and sentinel. Documentation to enable asc for iot and sentinel is added in separate PR.

**Pull request merged**